plugin/evm/vm_upgrade_bytes_test.go

```diff
@@ -154,10 +154,10 @@ func TestVMUpgradeBytesNetworkUpgrades(t *testing.T) {
 
 	// Get a json specifying a Network upgrade at genesis
 	// to apply as upgradeBytes.
-	subnetEVMTimestamp := time.Unix(10, 0)
+	subnetEVMTimestamp := big.NewInt(0)
```

Comment:
subnetEVMTimestamp must be 0 to be activated at genesis

Reply:
Can we require that this is set directly in the genesis rather than in the upgrade bytes to make this simpler?